presentation/components/src/presentation-components/properties/InstanceKeyValueRenderer.tsx

```diff
@@ -23,6 +23,7 @@ import { useUnifiedSelectionContext } from "../unified-selection/UnifiedSelectio
 export class InstanceKeyValueRenderer implements IPropertyValueRenderer {
   public canRender(record: PropertyRecord) {
     return record.value.valueFormat === PropertyValueFormat.Primitive
+      && record.property.typename === StandardTypeNames.Navigation
       && (record.value.value === undefined || isInstanceKey(record.value.value));
   }
```

Comment:
This property value renderer was intended to be used with any property value that is an instance key. It should not matter whether it's a navigation property, or not.

Reply:
do you see a case where it would not be a navigation property value?

Reply:
reverted